internal/pgadmin/users.go

```diff
@@ -105,7 +137,57 @@ with create_app().app_context():
 
         db.session.add(user)
         db.session.commit()
-`
+
+        # Set the cluster and host name variable.
+        clustername = "%s"
+        hostname = "%s"
+
+        # Get or create the group as necessary
+        group = (
+            ServerGroup.query.filter_by(user_id=user.id).first() or
+            ServerGroup()
+        )
+        group.name = "Crunchy PostgreSQL Operator"
+        group.user_id = user.id
+        db.session.add(group)
+        db.session.commit()
+
+        # Get the existing server connection, if there is one.
+        oldserver = Server.query.filter_by(
+            servergroup_id=group.id,
+            user_id=user.id,
+        ).first()
```

Comment:
For the server connection like with the server group, I was also unable to set an identifier that wasn't otherwise configurable by the user via the pgAdmin GUI. However, I believe a different approach compared to the server group makes sense for two reasons. First, due to how active connections are handled, we already need to delete and re-add our server connection when the configuration changes. Second, the server connections can be deleted by the user via the pgAdmin web GUI, meaning they won't be 'leftover' when they're no longer needed.

That being the case, I think we'll want to stop managing any server connection that's been modified directly by the user and add the server `name` as an additional query parameter. For example, if they update the server `name`, we'll create a new server connection with the original name and leave the modified connection in place. At that point, any changes (port, password) that are needed will have to be copied over to the old, modified server. Alternatively, it could just be deleted.

I'm going to revisit how the server connection updates are handled, per the comments below, but I think that approach will make sense overall. There is likely still a chain of events where we'd grab the 'wrong'' server connection, but it would essentially require creating a new server connection with the same name, then deleting the one PGO created. That may be worth revisiting at some point in the future, but I don't think it's a show-stopper at this point.